Logger/LogFactory.cs

```diff
@@ -1,11 +1,28 @@
-﻿namespace Logger
+﻿namespace Logger;
+
+public class LogFactory
 {
-    public class LogFactory
+    private string? _FilePath;
+
+    public string? FilePath { get => _FilePath; }
+
+    public BaseLogger? CreateLogger(string className)
     {
-        public BaseLogger CreateLogger(string className)
+        if (_FilePath is null)
         {
-            
             return null;
         }
+
+        FileLogger newFileLogger = new FileLogger(_FilePath!)
```

Comment:
```suggestion
        FileLogger newFileLogger = new FileLogger(_FilePath)
```
You shouldn't need the `!` operator here.